src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay.tsx

```diff
@@ -121,38 +150,38 @@ const StatusDisplay = ({ titleStatus = "Polygon", mutate, refresh, name, record,
   const { openModal, closeModal } = useModalContext();
   const contentStatus = (
     <Text variant="text-12-light" as="p" className="text-center">
-      {DescriptionStatusMap[titleStatus]} <b style={{ fontSize: "inherit" }}>{name}</b>?
+      {t(DescriptionStatusMap[titleStatus])} <b style={{ fontSize: "inherit" }}>{t(name)}</b>?
     </Text>
   );
   const contentRequest = (
     <Text variant="text-12-light" as="p" className="text-center">
-      {DescriptionRequestMap[titleStatus]} <b style={{ fontSize: "inherit" }}>{name}</b>?
+      {t(DescriptionRequestMap[titleStatus])} <b style={{ fontSize: "inherit" }}>{t(name)}</b>?
```

Comment:
FWIW, it's fine to leave this as-is, but for components that are admin-only, we don't need to wrap in translation.

Reply:
Thanks for your feedback! This component will be used in both admin and user interfaces, so we need to keep the translation wrapping to ensure it works correctly for both contexts.

Reply:
If it's going to be used in PD interfaces, I would move it out of the `src/admin` space to make that clear.

Reply:
Thanks for the feedback! I made a mistake in my previous comment. This component is indeed only used in admin sections. I'll go ahead and remove the translation wrapping as it is not needed in this context. Apologies for the confusion.